lib/semantics/symbol.h

```diff
@@ -364,6 +364,7 @@ class Symbol {
 
   bool isSubprogram() const;
   bool HasExplicitInterface() const;
+  bool IsSeparateMp() const;
```

Comment:
Is Mp a word from the standard or an abbreviation that could be spelled out?

Reply:
It is used in the grammar for _mp-subprogram-stmt_ but mostly it is spelled out.

I thought that `IsSeparateModuleProcedure` was too long, though it is not used much so maybe it's okay.

`Proc` is frequently used as an abbreviation. `Mod` is not except for in `ModFile`. So `IsSeparateModuleProc` would be okay.

Reply:
+1 IsSeparateModuleProc

Reply:
OK